tests/utils.go

```diff
@@ -1721,25 +1725,50 @@ func cleanNamespaces() {
 	}
 }
 
-func removeNamespaces() {
+func removeNamespaces() error {
 	virtCli, err := kubecli.GetKubevirtClient()
-	PanicOnError(err)
+	if err != nil {
+		return err
+	}
 
 	// First send an initial delete to every namespace
 	for _, namespace := range testNamespaces {
 		err := virtCli.CoreV1().Namespaces().Delete(namespace, nil)
 		if !errors.IsNotFound(err) {
-			PanicOnError(err)
+			if err != nil {
+				return err
+			}
 		}
 	}
 
-	// Wait until the namespaces are terminated
+	// Force namesapce deletion by removing Finalizers
+	for _, namespace := range testNamespaces {
+		currentNamespace, err := virtCli.CoreV1().Namespaces().Get(namespace, metav1.GetOptions{})
+		if err != nil {
+			log.Log.Errorf("Could not get namespace in order to remove Finalizers %v", err)
```

Comment:
If you were not able to get the namespace then you should exit here... you'll get a NPE.

Reply:
If a namespace could not be found at this time, it means it already deleted

Reply:
Now I see it, I will fix it in next commit